public/botchat.css

```diff
@@ -25,9 +25,9 @@
         background-color: #3e92d2;
     }
 
-    h1, h2, h3, p, ul {
-        margin: 0;
-        padding: 0;
+    h1, h2, h3, h4, p, ul, ol {

```

Comment:
Changing the margin and padding here is a temporary thing. There's a larger work item here to integrate the FormattedText control into cards.